tests/templates/test_subroutines/test_adder.py

```diff
@@ -120,7 +120,8 @@ def circuit(x):
             mod = 2 ** len(x_wires)
 
         # pylint: disable=bad-reversed-sequence
-        result = sum(bit * (2**i) for i, bit in enumerate(reversed(circuit(x))))
+        out = list(circuit(x)[0, :])
+        result = sum(bit * (2**i) for i, bit in enumerate(reversed(out)))
```

Comment:
```suggestion
        out = circuit(x)[0]
        result = sum(bit * (2**i) for i, bit in enumerate(reversed(out)))
```
Why won't this work as same? I ran it locally and it looks fine

Reply:
just a test so I don't suppose it matters much.